site/docs/samples/omd-discover-s3-asset.md

```diff
@@ -0,0 +1,61 @@
+# Discovering an S3 asset through the OpenMetadata UI
+
+This page explains how to discover an existing S3 asset through the OpenMetadata UI. The screenshots refer to `localstack` cloud storage, but the explanations also apply to other S3 services.
```

Comment:
maybe a link to the read sample where the there is an option to make a S3 object can be helpful here.
something like : "existing S3 asset (to add a s3 assest refer to the read...) through the OpenMetadata UI"